### DISCUSSION TITLE
Trouble opening multiple timeseries data with Xarray

### DISCUSSION THREAD

**Discussion by Hamadou867** [pydata/xarray#5423] [Q&A]
Dear all,
I am getting an error when trying to open many files with xarray. My code is:

import netCDF4
import numpy
import xarray as xr
import os
os.getcwd()
os.chdir("D:\THESIS_LMD_CNES_2020\Litterature_Review_LMD_CNES_2020\Synergie_PRI_SIF_GPP\ICOS_data_Flux\TROPO_SIF\TROPO_SIF683nm_2018-05")
data = xr.open_mfdataset('TROPO_redSIF_*.nc', decode_times =False, decode_cf = True, 
                         coords ='different',decode_timedelta = True, decode_coords = False)

The error notice: ValueError: Coordinate variable TIME is neither monotonically increasing nor monotonically decreasing on all datasets.
You can find an example of the data here in google drive: https://drive.google.com/drive/folders/1dTLT7q5l1fqycX12mGvtp0VwBJyi4t36?usp=sharing

Any help from this issue would be kindly appreciated.

Hamadou.

**Comment by max-sixty** [comment]
Please include the full stack trace, please try and format your example, and if possible reduce the error — linking to data adds overhead for others — e.g. try loading a single file.
